ros/src/computing/perception/localization/packages/orb_localizer/nodes/place_recognizer/place_recognizer.cpp

```diff
@@ -0,0 +1,395 @@
+/*
+ * place_recognizer.cpp
+ *
+ *  Created on: May 18, 2017
+ *      Author: sujiwo
+ */
+
+#include <iostream>
+#include <vector>
+#include <string>
+#include <ros/ros.h>
+#include <geometry_msgs/PoseStamped.h>
+#include <sensor_msgs/Image.h>
+#include <sensor_msgs/image_encodings.h>
+#include <cv_bridge/cv_bridge.h>
+#include <image_transport/image_transport.h>
+#include <boost/filesystem.hpp>
+#include <tf/tf.h>
+#include "System.h"
+#include "Map.h"
+#include "Frame.h"
+#include "ORBmatcher.h"
+#include "MapPoint.h"
+#include "PnPsolver.h"
+#include "ORBVocabulary.h"
+#include "ORBextractor.h"
+#include "Optimizer.h"
+#include "../common.h"
```

Comment:
Unix shortcuts are discouraged: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes.
You can path to common.h in https://github.com/CPFL/Autoware/blob/a23a6f25f403dd953c000fb463acf8c6fcedeffb/ros/src/computing/perception/localization/packages/orb_localizer/CMakeLists.txt#L74.